From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Darien Imai <darien@dpsi.jp>
Date: Sun, 9 May 2021 13:00:25 -0700
Subject: [PATCH] Disable sand duping fix.


diff --git a/src/main/java/net/minecraft/world/entity/item/FallingBlockEntity.java b/src/main/java/net/minecraft/world/entity/item/FallingBlockEntity.java
index 8336ea928faa92c6f58f8cdfb9faf1d8e26c9ccf..69280a3bae5333bacdbbfc47eb447b2143f5c352 100644
--- a/src/main/java/net/minecraft/world/entity/item/FallingBlockEntity.java
+++ b/src/main/java/net/minecraft/world/entity/item/FallingBlockEntity.java
@@ -106,11 +106,11 @@ public class FallingBlockEntity extends Entity {
 
     @Override
     public void tick() {
-        // Paper start - fix sand duping
-        if (this.isRemoved()) {
-            return;
-        }
-        // Paper end - fix sand duping
+//        // Paper start - fix sand duping
+//        if (this.isRemoved()) {
+//            return;
+//        }
+//        // Paper end - fix sand duping
         if (this.blockState.isAir()) {
             this.discard();
         } else {
