From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Darien Imai <darien@dpsi.jp>
Date: Sun, 9 May 2021 11:38:10 -0700
Subject: [PATCH] threereeds branding.


diff --git a/pom.xml b/pom.xml
index 1cd91deffb55e31f6a1f11af69b1fa3aa17abcf0..4bd2bc7043ea6cd96918058ea7698b600c58f7d8 100644
--- a/pom.xml
+++ b/pom.xml
@@ -39,8 +39,8 @@
 
     <dependencies>
         <dependency>
-            <groupId>com.mygroupid</groupId>
-            <artifactId>myairplanefork-api</artifactId>
+            <groupId>jp.dpsi</groupId>
+            <artifactId>threereeds-api</artifactId>
             <version>${project.version}</version>
             <scope>compile</scope>
         </dependency>
diff --git a/src/main/java/com/destroystokyo/paper/PaperVersionFetcher.java b/src/main/java/com/destroystokyo/paper/PaperVersionFetcher.java
index 3072ae4ab8fff7e53d1ff8b56fe762d14e3baea8..d759f2076f1e112e1c7a71014e10ed06af8e0cb2 100644
--- a/src/main/java/com/destroystokyo/paper/PaperVersionFetcher.java
+++ b/src/main/java/com/destroystokyo/paper/PaperVersionFetcher.java
@@ -29,8 +29,8 @@ public class PaperVersionFetcher implements VersionFetcher {
     @Nonnull
     @Override
     public Component getVersionMessage(@Nonnull String serverVersion) {
-        String[] parts = serverVersion.substring("git-MyAirplaneFork-".length()).split("[-\\s]"); // Tuinity // MyAirplaneFork
-        final Component updateMessage = getUpdateStatusMessage("MyGithubName/MyAirplaneFork", GITHUB_BRANCH_NAME, parts[0]); // Tuinity // MyAirplaneFork
+        String[] parts = serverVersion.substring("git-threereeds-".length()).split("[-\\s]"); // Tuinity // MyAirplaneFork // threereeds
+        final Component updateMessage = getUpdateStatusMessage("dpsi/threereeds", GITHUB_BRANCH_NAME, parts[0]); // Tuinity // MyAirplaneFork // threereeds
         final Component history = getHistory();
 
         return history != null ? TextComponent.ofChildren(updateMessage, Component.newline(), history) : updateMessage;
diff --git a/src/main/java/net/minecraft/server/MinecraftServer.java b/src/main/java/net/minecraft/server/MinecraftServer.java
index 0c013dd8e17e84e9c006d61bbc9d3ca830c1cd6c..227181773c4b921b484f0d4d1f2aabfc1ae8a884 100644
--- a/src/main/java/net/minecraft/server/MinecraftServer.java
+++ b/src/main/java/net/minecraft/server/MinecraftServer.java
@@ -1650,7 +1650,7 @@ public abstract class MinecraftServer extends IAsyncTaskHandlerReentrant<TickTas
     }
 
     public String getServerModName() {
-        return "MyAirplaneFork"; // MyAirplaneFork // Airplane // Tuinity //Paper - Paper > // Spigot - Spigot > // CraftBukkit - cb > vanilla!
+        return "threereeds"; // threereeds // MyAirplaneFork // Airplane // Tuinity //Paper - Paper > // Spigot - Spigot > // CraftBukkit - cb > vanilla!
     }
 
     public CrashReport b(CrashReport crashreport) {
diff --git a/src/main/java/org/bukkit/craftbukkit/CraftServer.java b/src/main/java/org/bukkit/craftbukkit/CraftServer.java
index 6263352e543449fe1f62b7576f9b896e96caec4f..71419133a755d2ca30b095f76b20223d27428a6f 100644
--- a/src/main/java/org/bukkit/craftbukkit/CraftServer.java
+++ b/src/main/java/org/bukkit/craftbukkit/CraftServer.java
@@ -230,7 +230,7 @@ import javax.annotation.Nullable; // Paper
 import javax.annotation.Nonnull; // Paper
 
 public final class CraftServer implements Server {
-    private final String serverName = "MyAirplaneFork"; // Paper // Tuinity // Airplane // MyAirplaneFork
+    private final String serverName = "threereeds"; // Paper // Tuinity // Airplane // MyAirplaneFork // threereeds
     private final String serverVersion;
     private final String bukkitVersion = Versioning.getBukkitVersion();
     private final Logger logger = Logger.getLogger("Minecraft");
diff --git a/src/main/java/org/bukkit/craftbukkit/util/Versioning.java b/src/main/java/org/bukkit/craftbukkit/util/Versioning.java
index 0e18f778a434b8b22bb595f06cc6ef7833e64489..49876329e1c8bef225765d45da87d3b3ec4013a8 100644
--- a/src/main/java/org/bukkit/craftbukkit/util/Versioning.java
+++ b/src/main/java/org/bukkit/craftbukkit/util/Versioning.java
@@ -11,7 +11,7 @@ public final class Versioning {
     public static String getBukkitVersion() {
         String result = "Unknown-Version";
 
-        InputStream stream = Bukkit.class.getClassLoader().getResourceAsStream("META-INF/maven/com.mygroupid/myairplanefork-api/pom.properties"); // Tuinity // Airplane
+        InputStream stream = Bukkit.class.getClassLoader().getResourceAsStream("META-INF/maven/jp.dpsi/threereeds-api/pom.properties"); // Tuinity // Airplane // threereeds
         Properties properties = new Properties();
 
         if (stream != null) {
