From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Darien Imai <darien@dpsi.jp>
Date: Sun, 9 May 2021 13:00:25 -0700
Subject: [PATCH] Disable sand duping fix.


diff --git a/src/main/java/net/minecraft/world/entity/item/FallingBlockEntity.java b/src/main/java/net/minecraft/world/entity/item/FallingBlockEntity.java
index 0c94b4cb6ee0c3dffe0b67a2291d0160ae0ef96f..106270fe7613da074f164926c3c55fb22a074722 100644
--- a/src/main/java/net/minecraft/world/entity/item/FallingBlockEntity.java
+++ b/src/main/java/net/minecraft/world/entity/item/FallingBlockEntity.java
@@ -110,11 +110,11 @@ public class FallingBlockEntity extends Entity {
 
     @Override
     public void tick() {
-        // Paper start - fix sand duping
-        if (this.isRemoved()) {
-            return;
-        }
-        // Paper end - fix sand duping
+//        // Paper start - fix sand duping
+//        if (this.isRemoved()) {
+//            return;
+//        }
+//        // Paper end - fix sand duping
         if (this.blockState.isAir()) {
             this.discard();
         } else if (this.level.isClientSide && this.removeAtMillis > 0L) {
