From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Darien Imai <darien@dpsi.jp>
Date: Mon, 26 Jul 2021 12:50:23 -0700
Subject: [PATCH] fixes


diff --git a/build.gradle.kts b/build.gradle.kts
index b7e5bdcfe04ab7d8d8d453a787849963640262a5..33b67a37d94784e87a10ab8bc87fb9aaf86a568e 100644
--- a/build.gradle.kts
+++ b/build.gradle.kts
@@ -34,7 +34,7 @@ repositories {
 }
 
 dependencies {
-    implementation(project(":Tuinity-API")) // Tuinity
+    implementation(project(":threereeds-API")) // Tuinity
     implementation("com.destroystokyo.paper:paper-mojangapi:1.16.5-R0.1-SNAPSHOT") // Tuinity
     // Paper start
     implementation("org.jline:jline-terminal-jansi:3.12.1")
@@ -86,7 +86,7 @@ tasks.jar {
         attributes(
             "Main-Class" to "org.bukkit.craftbukkit.Main",
             "Implementation-Title" to "CraftBukkit",
-            "Implementation-Version" to "git-Tuinity-$implementationVersion", // Tuinity
+            "Implementation-Version" to "git-threereeds-$implementationVersion", // Tuinity
             "Implementation-Vendor" to date, // Paper
             "Specification-Title" to "Bukkit",
             "Specification-Version" to project.version,
@@ -117,8 +117,8 @@ val patchReobfMappings by tasks.registering<PatchMappings> {
     inputMappings.set(generateReobfMappings.flatMap { it.reobfMappings })
     patch.set(rootProject.layout.cache.resolve("paperweight/upstreams/paper/build-data/reobf-mappings-patch.tiny"))
 
-    fromNamespace.set(Constants.DEOBF_NAMESPACE)
-    toNamespace.set(Constants.SPIGOT_NAMESPACE)
+    fromNamespace.set(io.papermc.paperweight.util.Constants.DEOBF_NAMESPACE)
+    toNamespace.set(io.papermc.paperweight.util.Constants.SPIGOT_NAMESPACE)
 
     outputMappings.set(layout.cache.resolve("paperweight/mappings/reobf-patched.tiny"))
 }
@@ -182,7 +182,7 @@ abstract class IncludeMappings : BaseTask() {
 }
 
 val includeMappings = tasks.register<IncludeMappings>("includeMappings") {
-    inputJar.set(tasks.fixJarForReobf.flatMap { it.outputJar })
+    inputJar.set(tasks.shadowJar.flatMap { it.archiveFile }) // Airplane
     mappings.set(tasks.reobfJar.flatMap { it.mappingsFile })
 }
 
